Demo/Demo/SwiftUIComponents/SwiftUINativeCheckoutDemo.swift

```diff
@@ -34,29 +38,45 @@ struct SwiftUINativeCheckoutDemo: View {
         NavigationView {
             ZStack {
                 VStack(spacing: 16) {
-                    Picker("Checkout ", selection: $checkoutTypeSelection) {
-                        ForEach(CheckoutType.allCases) {type in
-                            Text(type.rawValue)
-                        }
+                    VStack {
+                        Form {
+                            Picker("Checkout ", selection: $checkoutTypeSelection) {
+                                ForEach(CheckoutType.allCases) {type in
+                                    Text(type.rawValue)
+                                }
+                            }
+                        }.navigationBarTitle("", displayMode: .inline)
```

Comment:
```suggestion
                        }
```

Can we just omit this if we don't want to set a title?

Reply:
same here, its a way of reducing space that `Form` takes.